WcaOnRails/spec/lib/relations_spec.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+require 'relations'
+
+RSpec.describe "Relations" do
+  before do
+    allow(Relations).to receive(:linkings).and_return(
```

Comment:
I'm not wild about mocks when an alternative would do. Couldn't we just populate the `linkings` table here instead?